### ISSUE TITLE
Find a realtime implementation of optimal transport

### ISSUE THREAD

**Issue by jamesb93** [flucoma/learn-website#311]
The learn article for version 1.0 likely wont have a realtime version, therefore this is something that can be improved in the future.

**Comment by michal-cab** [comment]
yes! - it would be really very nice to have some tutorial for this - e.g. how to do realtime matching of input audio stream to ordered FluidDataSet.
